README.md

```diff
@@ -40,6 +55,26 @@ features and fixes, directly. Although it is reasonably stable, you are more lik
 breaking changes when running the `devel` branch. We recommend getting involved
 in the Ansible community if you want to run the `devel` branch.
 
+## Troubleshooting
+
+If you face any problems while trying to install Ansible here is how you can proceed:
```

Comment:
Like FAQ, this should be a link to a much more comprehensive document or even just the forums